client/src/components/Search/SearchByRepoView/SearchByRepoView.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import TextInput from '../TextInput/TextInput';
+import Dropdown from '../Dropdown/Dropdown';
+import RangeInput from '../RangeInput/RangeInput';
+
+const SearchByRepoView = ({ saveInput }) => {
+    //list of technologies avalible to search
+    const technologiesArr = ['JavaScript', 'Python', 'C#', 'COBOL'];
```

Comment:
These should be taken from the backend, please create a task for that. OK for now.

Reply:
Definitely. Task created. I have put it in backlog.